programs/ido-pool/src/lib.rs

```diff
@@ -81,20 +86,42 @@ pub mod ido_pool {
         }
         let pool_account = &mut ctx.accounts.pool_account;
         pool_account.start_ido_ts = start_ido_ts;
-        pool_account.end_deposits_ts = end_deposits_ts;
         pool_account.end_ido_ts = end_ido_ts;
         pool_account.withdraw_melon_ts = withdraw_melon_ts;
         Ok(())
     }
 
-    #[access_control(unrestricted_phase(&ctx))]
+    pub fn modify_max_usdc_tokens(
```

Comment:
```suggestion
    #[access_control(future_start(ctx.accounts.pool_account.start_ido_ts))]
    pub fn modify_max_usdc_tokens(
```
to make sure its only changeable before the ido starts?